nohup: ignoring input
Traceback (most recent call last):
  File "/data/chy/online/ImageNet/test_idefics.py", line 10, in <module>
    model = IdeficsForVisionText2Text.from_pretrained(checkpoint, torch_dtype=torch.bfloat16).to(device)
  File "/home/chy63/.conda/envs/openflamingo/lib/python3.9/site-packages/transformers/modeling_utils.py", line 2389, in from_pretrained
    resolved_config_file = cached_file(
  File "/home/chy63/.conda/envs/openflamingo/lib/python3.9/site-packages/transformers/utils/hub.py", line 429, in cached_file
    resolved_file = hf_hub_download(
  File "/home/chy63/.local/lib/python3.9/site-packages/huggingface_hub/utils/_validators.py", line 106, in _inner_fn
    validate_repo_id(arg_value)
  File "/home/chy63/.local/lib/python3.9/site-packages/huggingface_hub/utils/_validators.py", line 154, in validate_repo_id
    raise HFValidationError(
huggingface_hub.errors.HFValidationError: Repo id must be in the form 'repo_name' or 'namespace/repo_name': '/data/share/pyz/model_weight/idefics-9b'. Use `repo_type` argument if needed.
